From 89e40fb26cae3e3810015add53e7d8220f055afa Mon Sep 17 00:00:00 2001
From: Andreas Schmidt <andreas.schmidt@wago.com>
Date: Thu, 19 Jul 2018 14:12:17 +0200
Subject: [PATCH] omap: fix compatible conditional calls for wago,pfc200

In commit a88f655c29213a9c2885d9697f268b4c31335ea0 was compatible
string "wago,pfc200" to "wago,am3505-pfc" renamed. But conditions
in mach-omap2/io.c and plat-omap/counter_32k.c were not adapted.
This patch adapt the conditions to new "wago,am3505-pfc" compatible
string.

Signed-off-by: Peter Galka <peter.galka@wago.com>
Signed-off-by: Andreas Schmidt <andreas.schmidt@wago.com>
---
 arch/arm/mach-omap2/io.c         | 7 ++++---
 arch/arm/plat-omap/counter_32k.c | 2 +-
 2 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-omap2/io.c b/arch/arm/mach-omap2/io.c
index 1d4b565..c0059da 100644
--- a/arch/arm/mach-omap2/io.c
+++ b/arch/arm/mach-omap2/io.c
@@ -560,9 +560,10 @@ void __init am35xx_init_early(void)
 void __init omap3_init_late(void)
 {
 	omap_common_late_init();
-	if (!of_machine_is_compatible("wago,pfc200")) {
-			omap3_pm_init();
-	}
+
+	if (!of_machine_is_compatible("wago,am3505-pfc"))
+		omap3_pm_init();
+
 	omap2_clk_enable_autoidle_all();
 }
 
diff --git a/arch/arm/plat-omap/counter_32k.c b/arch/arm/plat-omap/counter_32k.c
index d392dae..57fb9b2 100644
--- a/arch/arm/plat-omap/counter_32k.c
+++ b/arch/arm/plat-omap/counter_32k.c
@@ -107,7 +107,7 @@ int __init omap_init_clocksource_32k(void __iomem *vbase)
          * We use that one. This should give us a drifting of appr. 14 ppm ~ 7.5 min/year.
          *
          */
-	if (of_machine_is_compatible("wago,pfc200"))
+	if (of_machine_is_compatible("wago,am3505-pfc"))
 		omap2_32ksyncnt = 32499;
 
 	/*
-- 
2.7.4

